packages/itwinui-react/src/core/Toast/Toaster.tsx

```diff
@@ -18,13 +15,13 @@ export type ToasterSettings = {
 
    * When `placement` is set to a top value, order defaults to 'descending', otherwise 'ascending'.
    */
-  order?: 'descending' | 'ascending';
+  order: 'descending' | 'ascending';
```

Comment:
Both setting props are mandatory now?

Reply:
no, it's using `Partial` in the dispatch action (line 165)